pages/about.html

```diff
@@ -18,7 +18,8 @@
   <header>
     <section>
       <a href="index.html">
```

Comment:
There is a bug in the app right now where if you are on the AboutUs page and click the logo to go back to the home page, it gives an error. `Cannot GET /pages/index.html`

I believe change this `a` tag `href` to `../index.html` might fix it?

Reply:
This bug also happens when clicking the logo from the `shoppinglist.html` page as well